zabbix_metrics_watcher.py

```diff
@@ -214,14 +217,41 @@ def run(zabbix_host, osbs_master, config, instance):
 
             logger.info("Found build %s in %s", build_name, status)
             build = Build(build_name, cmd_base)
-            if status == 'Pending':
+            if status == 'New':
+                now = datetime.datetime.now()
+                if build_name not in builds_in_new:
+                    builds_in_new[build_name] = now
+                spent_in_new = [(now - start).total_seconds() for start in builds_in_new.values()]
```

Comment:
This should be measured also when the builds are still in New. Luckily Openshift seems to send updates about every minute that the build is still in New, so this var will be updated.